features/us55-borrower-verified-landing-page.feature

```diff
@@ -0,0 +1,18 @@
+@us55
+
+Feature: Check content on Borrower Landing Page
+
+Acceptance Criteria
+    Page content matches wireframe
+
+Scenario Outline: Borrower arrives landing Page
+    Given I reach the Borrower Identity Verified Page
+    Then I should see <Link_button>
+    And the following <Texts>
+
+    Examples:
+      | Link_button            | Texts                                     |
+      | "Continue to mortgage" | "Read your mortgage details"              |
+      | ""                     | "Request a code"                          |
+      | ""                     | "Use the code to sign your mortgage deed" |
+      | ""                     | "don't need to sign a paper document"     |
```

Comment:
I would leave the Then statement out of the scenario outline and just pass through the link text in the step, only as currently you're executing that step three times without actually checking anything with it because there is only one button to check. I would keep the And statement to use table though thats fine as there are multiple paragraphs to check on the page.